go.mod

```diff
@@ -37,7 +37,7 @@ require (
 	github.com/filecoin-project/sector-storage v0.0.0-20200805173933-deec7a2658d4
 	github.com/filecoin-project/specs-actors v0.8.7-0.20200805174427-9d42fb163883
 	github.com/filecoin-project/specs-storage v0.1.1-0.20200622113353-88a9704877ea
-	github.com/filecoin-project/storage-fsm v0.0.0-20200805013058-9d9ea4e6331f
+	github.com/filecoin-project/storage-fsm v0.0.0-20200810142921-b9bc6718b0b2
```

Comment:
This points to a PR branch: https://github.com/filecoin-project/storage-fsm/pull/54. Needs to be updated when the PR is merged.